test/tensors.jl

```diff
@@ -126,6 +126,11 @@ for V in spacelist
                         @test t === @constinferred TensorMap(t.data, W)
                     end
                 end
+                for T in (Int, Float32, ComplexF64)
+                    t = randn(T, W1 ⊗ W2, typeof(W1)())
```

Comment:
`W1`, `W2`, ... are already `HomSpace` objects; I guess you wanted `V1` and `V2`. 

Should we have a `zero(V)` to replace `typeof(V)()`? Since we don't really use `+` for the direct sum of spaces, I am not sure if recycling `zero` is really appropriate. Then again:
<img width="726" alt="Screenshot 2024-11-20 at 09 30 55" src="https://github.com/user-attachments/assets/67a4c4a5-17d5-4e62-885a-07fc4632e7c4">

Reply:
Yeah, I noticed that I messed up, I tried fixing this just before I left the office yesterday. I'll fix it later today, and add the zero, since it's also just more practical